x-pack/plugins/ml/public/data_frame/components/group_by_list/group_by_label_summary.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('Data Frame: <GroupByLabelSummary />', () => {
       field: 'the-group-by-field',
       aggName: 'the-group-by-agg-name',
       dropDownName: 'the-group-by-drop-down-name',
-      interval: '10m',
+      calendar_interval: '10m',
```

Comment:
As above. seeing as you can only specify `1m`, `1h` etc in the UI, would `1m` rather than `10m` be a better test?

Reply:
Fixed in 71c2346.